services/funfuzz/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM mozillasecurity/fuzzos:latest
+
+LABEL maintainer Gary Kwong <gkwong@mozilla.com>
+WORKDIR /home/worker/funfuzz
+
+USER root
+COPY recipes /tmp/recipes
+RUN /tmp/recipes/install_prerequisites.sh \
+    && rm -rf /tmp/recipes/install_prerequisites.sh \
+    && chown -R worker:worker /home/worker
+
+ENV USER      worker
+ENV HOME      /home/worker
+ENV LOGNAME   worker
+ENV HOSTNAME  worker
+ENV LANG      en_US.UTF-8
+ENV LC_ALL    en_US.UTF-8
+
+ENV PATH="${HOME}/.cargo/bin:${HOME}/.local/bin:${PATH}"
+ENV LD_LIBRARY_PATH=.
```

Comment:
I am not sure this could cause problems if it is defined globally rather than in a script where it is to be used.

Reply:
True, this needs to be added whenever a user chooses to get into the funfuzz container. Any ideas?

Reply:
You could also add those things to some sort of init script. 
Example:

Dockerfile
```
CMD = [funfuzz.sh]
```

funfuzz.sh
```
export bash envs
./run_fuzzer
```